docs/index.md

```diff
@@ -1,23 +1,24 @@
 ---
-title: Overview
+title: Apify CLI overview
+description: An introduction to Apify CLI, a command-line interface for creating, developing, building, and running Apify Actors and managing the Apify cloud platform.
+sidebar_label: Overview
 ---
 
-Apify command-line interface (Apify CLI) helps you create, develop, build and run
-[Apify Actors](https://apify.com/actors),
-and manage the Apify cloud platform from any computer.
+Apify command-line interface (Apify CLI) helps you create, develop, build and run [Apify Actors](https://apify.com/actors), and manage the Apify platform from any computer.
 
-Apify Actors are cloud programs that can perform arbitrary web scraping, automation or data processing job.
-They accept input, perform their job and generate output.
-While you can develop Actors in an online IDE directly in the [Apify web application](https://console.apify.com/),
-for complex projects it is more convenient to develop Actors locally on your computer
-using <a href="https://github.com/apify/apify-sdk-js">Apify SDK</a>
-and only push the Actors to the Apify cloud during deployment.
-This is where the Apify CLI comes in.
+Apify Actors are cloud programs that can perform arbitrary web scraping, automation or data processing job. They accept input, perform their job and generate output. The Apify CLI enables you to develop Actors locally on your computer using your preferred code editor, version control system, and development tools. This approach gives you full control over your development environment, makes it easier to work on complex projects, and allows you to leverage the [Apify SDK](https://github.com/apify/apify-sdk-js) with all its powerful features. When you're ready, simply push your Actor to the Apify platform for deployment.
```

Comment:
To be honest, this is quite a wall of text. Can we somehow break it up to make it more easily consumable?